Tests/CasePathsMacrosTests/CasePathableMacroTests.swift

```diff
@@ -968,4 +968,102 @@ final class CasePathableMacroTests: XCTestCase {
       """#
     }
   }
+
+  func testComments() {
+    assertMacro {
+      """
+      @CasePathable enum Foo {
+        // Comment above case
+        case bar
+        /*Comment before case*/ case baz(Int)
+        case fizz(buzz: String)  // Comment on case
+        case fizzier/*Comment in case*/(Int, buzzier: String)
+      }
+      """
+    } expansion: {
+      #"""
+      enum Foo {
+        // Comment above case
+        case bar
+        /*Comment before case*/ case baz(Int)
+        case fizz(buzz: String)  // Comment on case
+        case fizzier/*Comment in case*/(Int, buzzier: String)
+
+        public struct AllCasePaths: Sequence {
+          public subscript(root: Foo) -> PartialCaseKeyPath<Foo> {
+            switch root {
+            case .bar:
+              return \.bar
+            case .baz:
+              return \.baz
+            case .fizz:
+              return \.fizz
+            case .fizzier/*Comment in case*/:
```

Comment:
we could even trim this comment, but it's also not hurting anything and just an edge case